src/main/java/org/apache/sling/commons/html/impl/parser/ParseException.java

```diff
@@ -0,0 +1,187 @@
+/* Generated By:JavaCC: Do not edit this line. ParseException.java Version 5.0 */

```

Comment:
Do we need to commit the generated code to source control? I think there are options to generate it at build time using Maven and then included in the packaged jar, without adding it to source control.

Reply:
Two things here, first code generation should be part of the build process. When I moved the code into this bundle I missed that plugin and I have since added it.

Yes. If you don't commit your generated code to source control, then there's no guarantee that you will be able to regenerate that exact code. Although maven is awesome compared to what it was before, I have seen jars updated on the backend without changing the version, I've all seen jars disappear. Although I trust maven central a little bit more then an independent repository. I believe in erring on the side of caution.

I also find it easier to troubleshoot with an IDE if the generated code is in the source directory.